config/v1/types_infrastructure.go

```diff
@@ -478,6 +478,32 @@ type AzurePlatformStatus struct {
 	// armEndpoint specifies a URL to use for resource management in non-soverign clouds such as Azure Stack.
 	// +optional
 	ARMEndpoint string `json:"armEndpoint,omitempty"`
+
+	// resourceTags is a list of additional tags to apply to Azure resources created for the cluster.
+	// See https://docs.microsoft.com/en-us/rest/api/resources/tags for information on tagging Azure resources.
+	// Azure supports a maximum of 50 tags per resource except for few, which have limitation of 15 tags.
```

Comment:
I wonder if we even bother mentioning the 50 limit here? We could just say, `Due to limitations on certain (maybe we could enumerate?) Azure resources, a maximum of 15 tags may be applied. OpenShift reserves 5 tags for internal use, allowing 10 tags for user configuration.`

Reply:
comment incorporated.